### PR TITLE
[Proposal] Cycatz-nycu's proposal

### PR DESCRIPTION
**Pull request by Cycatz-nycu**
---
A good PR should include the following parts:

- [X] A clear title
- [X] A meaningful message for PR, as well as its commits
- [X] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [X] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: proposal/Cycatz-nycu/README.org [hunk 1]
```diff
@@ -0,0 +1,83 @@
+* Uniramp 
+** Basic Information
+Uniramp is a tool for analyzing the gray values of glyphs in a given font; it also provides the image-to-Unicode conversion as a showcase.
```
Comment (yungyuc) [inline proposal/Cycatz-nycu/README.org:3]: Do you plan to make it interactive or batch processing?

File: proposal/Cycatz-nycu/README.org [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+However, most of the "Image to text" conversion tools online do not support custom character sets and Unicode codepoints. They usually implement it with the standard character ramp as mentioned above since it does not require additional character and font information. It's so bad that it does not fully utilize over 100000 code points defined in Unicode.   
+
+Therefore, this project aims to generate a Unicode-version art with the given image and font file. And the user can specify their own character set, which might contain CJK or emojis characters, to indicate the generated text art must only contain these characters. Furthermore, this program also displays the character ramp with each glyph's "coverage", which is defined as ~coverage = area of the shape / (glyph advance width * units per em)~, for doing further analysis.
```
Comment (yungyuc) [inline proposal/Cycatz-nycu/README.org:13]: AA is very useful for code remarks.  I think an optional ASCII mode makes a lot of sense when doing the unicode drawing.

File: proposal/Cycatz-nycu/README.org [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+ 
+** Perspective users 
++ Geeks: who want everything can be displayed in the terminal
```
Comment (yungyuc) [inline proposal/Cycatz-nycu/README.org:16]: You have thought about it :)

**Review by yungyuc — APPROVED** [review]
I look forward to reading your code.

**Comment by yungyuc** [comment]
@Cycatz-nycu I failed to notice that you did not include the github link to your repository.  Please use a separate PR to add the information before the deadline.

**Comment by Cycatz-nycu** [comment]
I have submitted another PR to add the project link: https://github.com/yungyuc/nsdhw_21au/pull/210

**Pull request merged**